deeprankcore/dataset.py

```diff
@@ -779,27 +776,35 @@ def load_one_graph(self, fname: str, entry_name: str)  -> Data: # pylint: disabl
         with h5py.File(fname, 'r') as f5:
             grp = f5[entry_name]
 
-            if (self.features_transform) and ('all' in self.features_transform):
-                transform = self.features_transform.get('all', {}).get('transform')
-                standard = self.features_transform.get('all', {}).get('standardize')
-            else:
-                transform = False
-                standard = False
             # node features
             if len(self.node_features) > 0:
                 node_data = ()
                 for feat in self.node_features:
+                    
+                    # resetting transformation and standardization for each feature
+                    transform = None
+                    standard = None
+
                     if feat[0] != '_':  # ignore metafeatures
                         vals = grp[f"{Nfeat.NODE}/{feat}"][()]
-                        
                         # get feat transformation and standardization
-                        if (self.features_transform is not None) and (feat in self.features_transform):
-                            transform = self.features_transform.get(feat, {}).get('transform')
-                            standard = self.features_transform.get(feat, {}).get('standardize')
+                        if (self.features_transform is not None):
+                            transform = self.features_transform.get('all', {}).get('transform')
+                            standard = self.features_transform.get('all', {}).get('standardize')
+                            # if no transformation is set for all features, check if one is set for the current feature
+                            if (transform is None) and (feat in self.features_transform):
+                                transform = self.features_transform.get(feat, {}).get('transform')
+                            # if no standardization is set for all features, check if one is set for the current feature
+                            if (standard is None) and (feat in self.features_transform):
+                                standard = self.features_transform.get(feat, {}).get('standardize')
 
                         # apply transformation
                         if transform:
-                            vals = transform(vals)
+                            with warnings.catch_warnings(record=True) as w:
+                                vals = transform(vals)
+                                if (len(w) > 0):
+                                    raise ValueError(f"Invalid value occurs when applying {transform} for feature {feat}."
+                                                    f"Please change the transformation function for {feat}.")
```

Comment:
I am currently unsure whether this will crash the entire run or just the entry.
Either way, I think it would be nice to state which entry this is occurring on, so that users can troubleshoot potential problems in the data rather than in the transformation.

Reply:
Right now it crashes the entire run. Are we okay with that? I modified the message as you suggested.